src/components/Button/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import "./button.scss";
 import clsx from "clsx";
 
-const Button = ({ text, className, onClick }) => (
-  <button className={clsx(className)} onClick={onClick}>
+const Button = ({ text, className, onClick, type }) => (
+  <button className={clsx(className)} onClick={onClick} type={type}>
```

Comment:
here you have props with the same name as attribute for a button. You can avoit it
const Button = ({ text, ...otherProps}) => (
<button {...otherProps}>.....